src/rust/src/lazy/dsl.rs

```diff
@@ -87,6 +87,7 @@ impl Expr {
 
         match (rtype, rlen) {
             (Rtype::Null, _) => Ok(dsl::lit(pl::NULL)),
+            (Rtype::Raw, _) => Ok(dsl::lit(robj_to_binary_vec(robj)?)), // Raw in R is seen as a vector of bytes, in polars it is a Literal, not wrapped in a Series.
```

Comment:
before merge, R Raw to Series[binary], literal-bianary to Series[binary] , Series[binary] to R list(Raw), and literal-binary to R Raw should also be implemented.

... and some round trip unit tests